docs/command-line/index.mdx

```diff
@@ -0,0 +1,251 @@
+import Admonition from "@theme/Admonition";
+
+# Интерфейс командной строки {#cli}
```

Comment:
I suggest CLI or Testplane CLI instead, subjectively my eyes are trained to look for "CLI" on the page

Same goes for the en version